### PR TITLE
Fix expected neighbor output when multiple ports connect to same neig…

### PR DESCRIPTION
**Pull request by lguohan**
…hbor

Previous code only enumerate on distinct neighbors, thus only one
port is shown for a neighbor.

Fix Azure/sonic-buildimage#5526

Signed-off-by: Guohan Lu <lguohan@gmail.com>

**- What I did**
Previous code only enumerate on distinct neighbors, thus only one
port is shown for a neighbor.

**- How I did it**
- Fix expected neighbor output when multiple ports connect to same neighbor
- Add unit test 
- Add fixture to select t1 topology database
- Introduce config/state/appl/asic/counter db for t1 topology

**- How to verify it**
Unit test and real dut

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: show/interfaces/__init__.py [hunk 1]
```diff
@@ -249,25 +249,29 @@ def expected(db, interfacename):
...
+            try:
+                device = neighbor_dict[port]['name']
+                body.append([port,
+                             device,
+                             device2interface_dict[device]['neighborPort'],
+                             neighbor_metadata_dict[device]['lo_addr'],
+                             neighbor_metadata_dict[device]['mgmt_addr'],
+                             neighbor_metadata_dict[device]['type']])
+            except KeyError:
+                pass
```
Comment (lguohan) [inline show/interfaces/__init__.py:274]: ignore port without neighbor device info.

File: tests/mock_tables/t1/counter_db.json [hunk 2]
```diff
@@ -0,0 +1,13890 @@
+{
```
Comment (neethajohn) [inline tests/mock_tables/t1/counter_db.json:1]: This is flex counter db

Comment (lguohan) [reply tests/mock_tables/t1/counter_db.json]: thanks. uploaded the correct one.

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
test failure is due to a different issue

**Pull request merged**